packages/ui/src/models/visualization/flows/camel-rest-visual-entity.ts

```diff
@@ -66,31 +75,21 @@ export class CamelRestVisualEntity extends AbstractCamelVisualEntity<{ rest: Res
     const schema = CamelCatalogService.getComponent(CatalogKind.Entity, 'rest');
 
     return {
-      definition: Object.assign({}, this.rest),

```

Comment:
This was preventing the configuration form from loading because we were assigning the restDef instead of the rest property.